### PR TITLE
Add controller and view

### PR DESCRIPTION
**Pull request by lynnette-orr**

### CODE REVIEW

File: server/routes/overview/overviewControllerV2.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    private readonly inductionService: InductionService,
+    private readonly educationAndWorkPlanService: EducationAndWorkPlanService,
+  ) {}
+
+  getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
+    const { prisonerSummary } = req.session
+
+    try {
+      const [inductionExists, allFunctionalSkills, prisonerGoals] = await Promise.all([
```
Comment (nathanrussell-moj-digital) [inline server/routes/overview/overviewControllerV2.ts:21]: Nice use of `Promise.all` 👍

File: server/routes/overview/overviewView.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  prisonerSummary: PrisonerSummary
+  functionalSkills: FunctionalSkills
+  inPrisonCourses: InPrisonCourseRecords
+  isPostInduction: boolean
+  lastUpdatedBy: string | null
+  lastUpdatedDate: Date | null
+  lastUpdatedAtPrisonName: string | null
+  noGoals: boolean
+  goalCounts: { activeCount: number; archivedCount: number; completedCount: number }
+  completedCourses: Array<InPrisonCourse>
```
Comment (nathanrussell-moj-digital) [inline server/routes/overview/overviewView.ts:16]: Do we need the view to know about the prisonNumber? (not sure any of our other view classes do they? 🤔 ) and also the separation of the different course types (completed, in-progress, withdrawn etc)
Why do we need those? 🤔

**Review by nathanrussell-moj-digital — COMMENTED** [review]
This is looking good 👍 

I think we want a unit test for the new controller class; plus I've left a comment on the view class re: I'm not sure if we need all those fields do we? 🤔

File: server/routes/overview/overviewView.ts (hunk 2 above)
Comment (lynnette-orr) [reply server/routes/overview/overviewView.ts]: Removed them.

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**